rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -503,7 +503,11 @@ void emu_settings::EnhanceSpinBox(QSpinBox* spinbox, SettingsType type, const QS
 	spinbox->setRange(min, max);
 	spinbox->setValue(val);
 
+#if QT_VERSION >= QT_VERSION_CHECK(5,14,0)
```

Comment:
This is just bad practise. We only do version checks if it is necessary

Reply:
I'll accept it in this case.
But you shouldn't need an overload afaik.
So
```suggestion
Connect(spinbox, &QSpinBox::textChanged
```
Should be correct in both cases